src/HttpWorker.php

```diff
@@ -157,6 +157,8 @@ private function hydrateRequest(Request $request, array $context): void
         $request->cookies = (array)($context['cookies'] ?? []);
         $request->uploads = (array)($context['uploads'] ?? []);
         $request->parsed = (bool)$context['parsed'];
+
+        $request->attributes['_parsed_body'] = $request->parsed;
```

Comment:
Attributed value looks better 👍 

1. The literal should be replaced with a public constant
2. I think the key should contain some prefix like `rr_` or `roadrunner`. @butschster WDYT?

Reply:
@roxblnfk, PR adjusted. :wink: